testing/benchmark/README.md

```diff
@@ -65,7 +65,8 @@ The main commands are:
   - `BENCHMARK_TIME`: Set the amount of time to run each benchmark scenario for. Defaults to `2m`.
```

Comment:
update the new benchmark warmup time default

Reply:
oh, forget about this. The testing/benchmark default didn't change. What changed was the github workflow setup only